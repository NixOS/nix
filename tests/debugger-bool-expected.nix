let 
  wat = arg : if ("wat") then arg else 10;
  foo = arg : arg2 : arg + arg2;
in
foo 12 (wat 11)