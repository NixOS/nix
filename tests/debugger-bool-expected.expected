[34;1m0[0m: [31;1merror: [0mvalue is [35;1ma string[0m while a Boolean was expected
[34;1m1[0m: while calling [35;1m'wat'[0m
[34;1m2[0m: while calling [35;1m'foo'[0m
[34;1m3[0m: while evaluating the file '[35;1m/home/bburdette/op-code/nix/tests/debugger-bool-expected.nix[0m':
